### PR TITLE
feat: adding cmek settings in log bucket

### PR DESCRIPTION
**Pull request by Saloni-Patidar**
Adding cmek_settings in log bucket.

### CODE REVIEW

File: modules/logbucket/main.tf [hunk 1]
```diff
@@ -40,6 +40,9 @@ resource "google_logging_project_bucket_config" "bucket" {
   enable_analytics = var.enable_analytics
   bucket_id        = var.name
   locked           = var.locked
+  cmek_settings {
```
Comment (imrannayer) [inline modules/logbucket/main.tf:43]: This code will be optional and should execute when keys are provided. Check [storage module](https://github.com/terraform-google-modules/terraform-google-log-export/blob/fd276c307245d76f4b136e00df60572458b19316/modules/storage/main.tf#L74) how to handle it?

Comment (Saloni-Patidar) [reply modules/logbucket/main.tf:43]: @imrannayer should we add a new folder with these settings as an option?

**Comment by Saloni-Patidar** [comment]
@imrannayer how can we solve this error?

**Comment by Saloni-Patidar** [comment]
@imrannayer are we good to merge this?

**Review by imrannayer — APPROVED** [review]

**Pull request merged**